docs/core/migration/20-21.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Migrate from NET Core 2.0 to 2.1
+description: Learn how to upgrade your .NET Core 2.0 app to 2.1.
+author: mairaw
+ms.author: mairaw
+ms.date: 06/15/2018
+---
+# Migrate from NET Core 2.0 to 2.1
+
+This article shows you how the basic steps for you to migrate your .NET Core 2.0 app to 2.1. If you're looking to migrate your ASP.NET Core application to 2.1, see [Migrate from ASP.NET Core 2.0 to 2.1](/aspnet/core/migration/20_21).
+
+For an overview of the new features in .NET Core 2.1, see [What's new in .NET Core 2.1](../whats-new/dotnet-core-2-1.md).
+
+## Update the project file to use 2.1 versions
+
+* Open the project file (the \*.csproj, \*.vbproj, or \*.fsproj file).
+
+* Change the [target framework](../../standard/frameworks.md) value from `netcoreapp2.0` to `netcoreapp2.1`. The target framework is defined on the `<TargetFramework>` or `<TargetFrameworks>` element.
+
+For example, change `<TargetFramework>netcoreapp2.0</TargetFramework>` to `<TargetFramework>netcoreapp2.1</TargetFramework>`.
+
+* Remove `<DotNetCliToolReference>` references for tools that are bundled in the .NET Core 2.1 SDK. These references include:
+
+  * [dotnet-watch](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-watch/README.md) (Microsoft.DotNet.Watcher.Tools)
+  * [dotnet-user-secrets](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-user-secrets/README.md) (Microsoft.Extensions.SecretManager.Tools)
+  * [dotnet-sql-cache](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-sql-cache/README.md) (Microsoft.Extensions.Caching.SqlConfig.Tools)
+  * [dotnet-dev-certs](https://github.com/aspnet/DotNetTools/blob/dev/src/dotnet-dev-certs/README.md) (?)
```

Comment:
nit: there is a question mark at the end

Reply:
yep, because I couldn't find how the reference to that was. 😄 I was hoping the reviewers would help me. @scottaddie do you know?

Reply:
@mairaw What's the question?

Reply:
I'm trying to add the exact references that need to be removed for each tool but I don't the one used for dotnet-dev-certs. also, can you check that the others are correct?

Reply:
I believe dotnet-dev-certs is something completely new that doesn't replace any references in the *.csproj* file.

@natemcmaster See Maira's question above. Can you please provide a list of *.csproj* references to be removed?

Reply:
Thanks @natemcmaster. The way it was written in the [.NET Core 2.1 blog post announcement](https://blogs.msdn.microsoft.com/dotnet/2018/05/30/announcing-net-core-2-1/), made me think that it existed before:

The following existing DotNetCliReferenceTool tools have been converted to in-box tools.

dotnet watch
dotnet dev-certs
dotnet user-secrets
dotnet sql-cache
dotnet ef
Remove project references to these tools when you upgrade to .NET Core 2.1.